autoprotocol/instruction.py

```diff
@@ -9,6 +9,7 @@
 
 # pragma pylint: disable=too-few-public-methods, redefined-builtin
 from .builders import *  # pylint: disable=unused-wildcard-import
+from .constants import *  # pylint: disable=unused-wildcard-import
```

Comment:
nit: if we know for sure we're only importing one object (`PROVISION_MEASUREMENT_MODES`) I think it would be worth explicitly importing it, and avoiding a wildcard import

Reply:
Followed previous import statement. Anyhow fixed it.